Homework16

JguyTestpro opened the pull request.

JguyTestpro committed:
Homework16

Berichicko commented:
You got this problem, please correct it. Please finish your homework.

![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/6956a606-27cb-4ef8-afb7-516a15ea8a94)

HINT

Test status: 
	
Test error (red). This status is assigned to tests that caused an exception from the tested source code.

For example

![Test error_en](https://user-images.githubusercontent.com/104761482/233709826-2179fc1f-6bb7-4df1-ad33-d65071cfa7b8.png)

JguyTestpro committed:
Update Homework16.java

This made sense after we went over it in the class last night, and the teacher told me why, what I had here was incorrect, and just the URL would be a  better choice.  Thanks Alex!
